### PR TITLE
Feature/count up timer

### PR DESCRIPTION
**Pull request by jonnyarndt**
Added a base factory class, unique countuptimer class, type, and joinmap. Tested existing countdowntimer and new countuptimer types on the bench. Also updated readme.md for countuptimer device.

### CODE REVIEW

File: src/TimerDeviceCountup.cs [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        /// </summary>
+        public void Start()
+        {
+            if (IsRunning) return;
+            IsRunning = true;
+            CountupTimerRunningFb.FireUpdate();
+
+            Debug.Console(1, this, "CountupTimer.Start() requested...");
+
+            if(_countupTimerTimeSpan == null)
```
Comment (ngenovese11) [inline src/TimerDeviceCountup.cs:72]: TimeSpans are structs, so this will never be null (structs are never null)

Comment (jonnyarndt) [reply src/TimerDeviceCountup.cs]: Commit [42518c4 ](https://github.com/PepperDash/epi-utilities-timer/commit/42518c4edff1f9d8ca654d93eeff439de855df30) removes the null check, adds Unit-Test logic for future reference, and updates the bridge IP ID to match Unit-Test.

**Comment by ngenovese11** [comment]
Please add the SPLWork folder to the git ignore and remove those files @jonnyarndt

**Comment by jonnyarndt** [comment]
@ngenovese11 - Any action for me on this?

**Review by jkdevito — APPROVED** [review]

**Pull request merged**